scan/lib/scan/runner.rb

```diff
@@ -39,11 +39,20 @@ def test_app
         }
       ]
       exit_status = 0
+
+      Scan.devices.each do |device|
+        app_identifier = Scan.config[:app_identifier]
+        app_identifier ||= UI.input("App Identifier: ")
```

Comment:
This `UI.input` would end up prompting for 1 to many times depending on the number of devices that are being iterated through here which probably isn't ideal. But I think this can be solved with the comment about combining the two new options into `reinstall_with_app_identifier` (mentioned above)

Reply:
This can be solved by me not being stupid 😂